app/views/govuk_publishing_components/components/docs/success_alert.yml

```diff
@@ -37,3 +37,9 @@ examples:
         orci. Proin semper porttitor ipsum, vel maximus justo rutrum vel.
         Morbi volutpat facilisis libero. Donec posuere eget odio non egestas.
         Nullam sed neque quis turpis.
+  with_custom_margin_bottom:
+    description: |
+      The component accepts a number for margin bottom from 0 to 9 (0px to 60px) using the [GOV.UK Frontend spacing scale](https://design-system.service.gov.uk/styles/spacing/#the-responsive-spacing-scale). The default margins for the component are inherited from the [notification-banner](https://github.com/alphagov/govuk-frontend/blob/main/package/govuk/components/notification-banner/_index.scss) styles defined in the Design System.
```

Comment:
Maybe amend this to say what the margin is? e.g. `By default the component has a bottom margin of 50px, inherited from...`

Reply:
I wondered about doing that but I didn't think it would be a good idea because the documentation would immediately go out of date if GOVUK Frontend update the spacing on this component at all. I thought linking to the `scss` file in `govuk-frontend` was a good compromise.